ui-kit/src/main/kotlin/org/p2p/uikit/organisms/UiKitSectionHeader.kt

```diff
@@ -0,0 +1,52 @@
+package org.p2p.uikit.organisms
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.widget.LinearLayout
+import androidx.core.view.isVisible
+import org.p2p.uikit.R
+import org.p2p.uikit.databinding.WidgetSectionHeaderBinding
+
+class UiKitSectionHeader @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : LinearLayout(context, attrs, defStyleAttr) {
+
+    private val binding = WidgetSectionHeaderBinding.inflate(LayoutInflater.from(context), this)
+
+    init {
+        context.obtainStyledAttributes(attrs, R.styleable.UiKitSectionHeader).use { typedArray ->
+            val withIcon = typedArray.getBoolean(R.styleable.UiKitSectionHeader_withIcon, true)
+            val defaultColor = context.getColor(R.color.text_mountain)
+            val tint = typedArray.getColor(R.styleable.UiKitSectionHeader_textIconTint, defaultColor)
+            val text = typedArray.getString(R.styleable.UiKitSectionHeader_sectionText)
+            val isCaps = typedArray.getBoolean(R.styleable.UiKitSectionHeader_isCaps, false)
+
+            binding.apply {
+                chevronImageView.isVisible = withIcon
+                titleTextView.isAllCaps = isCaps
+                titleTextView.text = text
+                titleTextView.setTextColor(tint)
+                chevronImageView.setColorFilter(tint)
+            }
+        }
+    }
+
+    fun setText(text: String) {
+        binding.titleTextView.text = text
+    }
+
+    fun showIcon(isVisible: Boolean) {
```

Comment:
i recommend to do this methods as properties - it would be fluent and consistent with Android std components

Reply:
or maybe `withImageOrGone` ?

Reply:
Due to description icon won't be changed so we should only manage it's visibility